pkg/controller/admissionchecks/provisioning/controller.go

```diff
@@ -267,6 +269,7 @@ func (c *Controller) syncOwnedProvisionRequest(ctx context.Context, wl *kueue.Wo
 					Parameters:            parametersKueueToProvisioning(prc.Spec.Parameters),
 				},
 			}
+			passProvReqParams(wl, req)
```

Comment:
What if the user modifies the annotations?
Is there a resync?

Reply:
Once ProvReq is created there is no way to modify it

Reply:
That's ok, but let's point it out in the documentation.